### DISCUSSION TITLE
Contribution: run examples against local changes

### DISCUSSION THREAD

**Discussion by arnaudbzn** [TanStack/query#4981] [Q&A] [answered]
`CONTRIBUTIONS.MD` file "Running examples" section explains how to run examples against local changes in watch mode.
But to make it works, I have to do the following change to my example `package.json`:
Before:
```
 "@tanstack/react-query": "^4.7.1",
```
After:
```
"@tanstack/react-query": "workspace:*",
```

Did I miss something?
If it's the way to go I can create an issue to update the `CONTRIBUTIONS.MD` file.

**Comment by TkDodo** [comment]
interesting - I didn't need to do that, it just works for me 🤔 

the problem is if we change it to `workspace:*`, the examples won't be runnable on codesandbox

**Comment by arnaudbzn** [comment]
> interesting - I didn't need to do that, it just works for me 🤔

If I strictly follow "Running examples" doc, it does not work for me :worried:
But happy to know that it should works without any tweaks on the `package.json` file to avoid to the issue with codesandbox.

I'll double check my config :pray:

**Comment by arnaudbzn** [comment, marked as answer]
Re-applying the repo setup solved the issue. Thank you!
